backend/fms_core/migrations/0058_v4_5_0.py

```diff
@@ -0,0 +1,160 @@
+                                                     ],
+        }
+        SUBPROTOCOLS_BY_PROTOCOL = {
+            "Axiom Experiment Run": ["Axiom: Denaturation and Hybridization",
+                                         "Axiom: GeneTitan Reagent Preparation",]
+        }
+        
+        
+        protocol_content_type = ContentType.objects.get_for_model(Protocol)
+
+        for protocol_name in PROPERTY_TYPES_BY_PROTOCOL.keys():
+            protocol = Protocol.objects.create(name=protocol_name,
+                                               created_by_id=admin_user_id, updated_by_id=admin_user_id)
+            reversion.add_to_revision(protocol)
+
+            for (property, value_type) in PROPERTY_TYPES_BY_PROTOCOL[protocol_name]:
+                pt = PropertyType.objects.create(name=property,
+                                                 object_id=protocol.id,
+                                                 content_type=protocol_content_type,
+                                                 value_type=value_type,
+                                                 is_optional=False,
+                                                 created_by_id=admin_user_id, updated_by_id=admin_user_id)
+                reversion.add_to_revision(pt)
+
+        # Attach subprotocols to parent protocol
+        for protocol_name in SUBPROTOCOLS_BY_PROTOCOL.keys():
+            parent = Protocol.objects.get(name=protocol_name)
+            for subprotocol_name in SUBPROTOCOLS_BY_PROTOCOL[protocol_name]:
+                child = Protocol.objects.get(name=subprotocol_name)
+                protocolbysubprotocol = ProtocolBySubprotocol.objects.create(child=child, parent=parent,
+                                                                             created_by_id=admin_user_id, updated_by_id=admin_user_id)
+                reversion.add_to_revision(protocolbysubprotocol)  
+
+        # Create Step
+        STEP = [
+            # {name, protocol_name}
+            {"name": "Axiom Experiment Run", "protocol_name": "Axiom Experiment Run", "expected_sample_type": SampleType.EXTRACTED_SAMPLE},
+        ]
+        
+        for step_info in STEP:
+            protocol = Protocol.objects.get(name=step_info["protocol_name"])
+
+            step = Step.objects.create(name=step_info["name"],
+                                       protocol=protocol,
+                                       expected_sample_type=step_info["expected_sample_type"],
+                                       created_by_id=admin_user_id,
+                                       updated_by_id=admin_user_id)
+            reversion.add_to_revision(step)
+        
+        # Create RunType Axiom
+        rt = RunType.objects.create(name="Axiom",
+                                    platform=platform,
+                                    protocol=protocol,
+                                    needs_run_processing=True,
```

Comment:
I don't think we'll get a run processing here just yet. As far as i can see this is not used anywhere ... but just to be safe let's set it to false for now.